### PR TITLE
Refactored filling catalogs to a command

### PR DESCRIPTION
**Pull request by joeribekker**
Changes were needed to make this work for changing catalogs. Also moved this to a command to prevent overhead in individual UPN record creation.

### CODE REVIEW

**Comment by joeribekker** [comment]
@dariomory if you could write a test for the new command, that would be great.

File: src/sdg/producten/management/commands/autofill.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        for catalog in ProductenCatalogus.objects.filter(
+            autofill=True, is_referentie_catalogus=True
+        ):
+            fields = [f.lower() for f in catalog.autofill_upn_filter]
+            if fields:
+                catalogs.append((catalog, fields))
+
+        # Bail early if no catalogs.
+        if not catalogs:
+            self.stdout.write("No catalogs found to autofill.")
```
Comment (dariomory) [inline src/sdg/producten/management/commands/autofill.py:30]: This should have a return

File: src/sdg/producten/management/commands/autofill.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            if upn.generieke_producten__count == 0:
+                generic_product = GeneriekProduct.objects.create(upn=upn)
+                LocalizedGeneriekProduct.objects.localize(
+                    instance=generic_product, languages=settings.SDG_ACTIVE_LANGUAGES
+                )
+                self.stdout.write(f'Created new generic product for "{upn}".')
+
+                generic_products = [generic_product]
+            else:
+                generic_products = GeneriekProduct.objects.filter(upn=upn)
```
Comment (dariomory) [inline src/sdg/producten/management/commands/autofill.py:47]:
```suggestion
            if upn.generieke_producten__count == 0:
                generic_product = GeneriekProduct.objects.create(upn=upn)
                LocalizedGeneriekProduct.objects.localize(
                    instance=generic_product, languages=TaalChoices.get_available_languages(),
                )
                self.stdout.write(f'Created new generic product for "{upn}".')
                generic_products = [generic_product]
            else:
                generic_products = upn.generieke_producten
```

As we already have TaalChoices to set the existing available languages it should be better to use it instead.

File: src/sdg/conf/base.py [hunk 3]
```diff
@@ -352,6 +352,8 @@
 ENVIRONMENT = config("ENVIRONMENT", "")
 SHOW_ALERT = True
 
+SDG_ACTIVE_LANGUAGES = ["nl", "en"]
+
```
Comment (dariomory) [inline src/sdg/conf/base.py:356]: Use `TaalChoices` instead.

**Review by dariomory — CHANGES_REQUESTED** [review]

**Review by dariomory — APPROVED** [review]

**Pull request merged**